react-router-test-app/app/root.tsx

```diff
@@ -1,4 +1,6 @@
-import '@flarelabs-net/jsx-localize/react';
+import { initializeJsxLocalize } from '@flarelabs-net/jsx-localize/react';
+initializeJsxLocalize();
```

Comment:
what sucks about this approach is that we'd have to invoke the initialization right here because we need the globals to be set before any other imported modules are initialized.

for this reason I'm not a huge fan of this change